docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-keycloak-oidc.md

```diff
@@ -156,3 +156,16 @@ When you fill the **Configure a Keycloak OIDC account** form and click on **Enab
 ### Keycloak Error: "Invalid grant_type"
 
   * In some cases, this error message may be misleading and is actually caused by setting the `Valid Redirect URI` incorrectly.
+
+### Unable to See Groups When Assigning Global Roles
+
+When setting up an Keycloak OIDC, if the user isn't part of any groups, `groupSearchEnabled` is disabled, preventing group search in Assign Global Roles. 
```

Comment:
```suggestion
If you use a user that is not part of any groups for initial setup, then you cannot search for groups when trying to assign a global role.
```

Some style-based suggestions and added some omissions:
- not mentioning keycloak since we're on the keyclocak page
- groupSearchEnabled since it's an implementation detail and not something they can current edit using normal procedures